packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin.tsx

```diff
@@ -264,7 +264,7 @@ export const TooltipPlugin = ({
   }
 
   return (
-    <Portal>
+    <Portal zIndexOffset={isActive ? 1 : 0}>
```

Comment:
I think this change may be impacting other places that we use `Portal` including on the `Dashboards` page.


https://github.com/grafana/grafana/assets/22381771/3d9aa1d1-b7fd-4714-82fc-fcb34ab48baa

We will probably want to do an audit of where we use this component and ensure that we aren't introducing regressions given the `Portal` component has a large surface area in the codebase

Reply:
Good catch, @nmarrs